tests/test_common.py

```diff
@@ -0,0 +1,12 @@
+from ccmlib.common import scylla_extract_mode
+
+
+def test_scylla_extract_mode():
+    assert scylla_extract_mode("build/dev") == 'dev'
+    assert scylla_extract_mode("../build/release") == 'release'
+    assert scylla_extract_mode("../build/release/scylla") == 'release'
+    assert scylla_extract_mode("/home/foo/scylla/build/debug") == 'debug'
+    assert scylla_extract_mode("url=../scylla/build/debug/scylla-package.tar.gz") == 'debug'
+    assert scylla_extract_mode("url=./scylla-x86_64-debug-package.tar.gz") == 'debug'
```

Comment:
I'm not sure about that...

I think the mode comes before the arch, e.g. http://downloads.scylladb.com.s3.amazonaws.com/unstable/scylla/branch-4.6/relocatable/2021-12-08T10%3A13%3A35Z/scylla-debug-aarch64-package.tar.gz

Reply:
Cc @yaronkaikov @benipeled

Reply:
Yup, ex. `scylla-debug-aarch64-package.tar.gz` or `scylla-debug-x86_64-unified-package-4.6.rc0.0.20211130.afc18d507.tar.gz`

PS> IMO the arch-name convention is wrong, should be at the end ex. `scylla-debug-package-aarch64.tar.gz` but that's another story.